### PR TITLE
Move build toolchain information to where it is needed

### PR DESCRIPTION
**Pull request by tpmccallum**
Signed-off-by: tpmccallum tim.mccallum@fermyon.com

Content must go through a pre-merge checklist.

## Pre-Merge Content Checklist

This documentation has been checked to ensure that:

- [x] The `title, `template`, and `date` are all set
- [x] Does this PR have a new menu item (anywhere in `templates/*.hbs` files) that points to a document `.md` that is set to publish in the future? If so please only publish the `.md` and `.hbs` changes in real-time (otherwise there will be a menu item pointing to a `.md` file that does not exist)
- [x] File does not use CRLF, but uses plain LF (hint: use `cat -ve <filename> | grep '^M' | wc -l` and expect 0 as a result) 
- [x] Has passed [`bart check`](https://developer.fermyon.com/bartholomew/quickstart)

Screenshot 2023-03-14 at 1 54 59 pm

- [x] Has run `npm run build-index`
- [x] Has been manually tested by running in Spin/Bartholomew (hint: use `PREVIEW_MODE=1` and run `npm run styles` to update styling)

Screenshot 2023-03-14 at 2 02 26 pm

- [x] Headings are using Title Case
- [x] Code blocks have the programming language set to properly highlight syntax and the proper copy directive
- [x] Have tested with `npm run test` and resolved all errors

Screenshot 2023-03-14 at 1 56 20 pm

### CODE REVIEW

File: content/spin/install.md [hunk 1]
```diff
@@ -108,6 +97,14 @@ $ cd spin && make build
 $ ./target/release/spin --help
 </code></pre>
 
+> Please note: On a fresh Linux installation, you will also need the standard build toolchain (`gcc`, `make`, etc.), the SSL library headers, and on some distributions you may need `pkg-config`. For example, on Debian-like distributions, including Ubuntu, you can install the standard build toolchain with a command like this:
```
Comment (itowlson) [inline content/spin/install.md:100]: This would also be needed for the Using Cargo to Install Spin method.

Comment (itowlson) [inline content/spin/install.md:100]: I would suggest rewording away from "a command like this" to "this command."

**Review by itowlson — CHANGES_REQUESTED** [review]
Just need to copy it to one more place I think

File: content/spin/install.md (hunk 1 above)
Comment (tpmccallum) [reply content/spin/install.md]: Done

Comment (tpmccallum) [reply content/spin/install.md]: I might have to re-word a little to cater for Windows users (as not to make it too Linux focussed when referring to cargo)

Comment (itowlson) [reply content/spin/install.md]: Ooh good catch

**Review by itowlson — APPROVED** [review]

File: content/spin/install.md (hunk 1 above)
Comment (tpmccallum) [reply content/spin/install.md]: As discussed, I agree, catering for the alternate platforms can go over to the contributing to Spin page, which is capable of diving into a bit more detail and providing good context. about installing from source.
